user/templates/templates.md

```diff
@@ -155,22 +155,24 @@ and also that it is not set as the default template.
 
 The procedure for uninstalling a template depends on how it was created.
 
-If the template was originaly created by cloning another template, then you can
+If the template was originally created by cloning another template, then you can
 delete it the same way as you would any other qube. In the Qube Manager,
 right-click on the template and select **Delete qube**. (If you're not sure,
 you can safely try this method first to see if it works.)
 
-If, on the other hand, the template came pre-installed or was installed by
-installing a template package in dom0, per the instructions
-[above](#installing), then you must execute the following type of command in
-dom0 in order to uninstall it:
+If, on the other hand, the template came pre-installed or was installed through
+a template package in dom0 using the instructions [above](#installing), then you
+must execute the following commands in dom0 in order to uninstall it.
```

Comment:
` template package` is pretty clumsy - most users wont know that they are using packages.
Why not just say - or was installed using `qvm-template` or the GUI tool  ? ( The installation section needs to be rewritten to reflect this.)